test/sqllogictest/table_func.slt

```diff
@@ -1710,3 +1710,33 @@ Source materialize.public.x
 Target cluster: quickstart
 
 EOF
+
+query T multiline
+EXPLAIN OPTIMIZED PLAN WITH (NO FAST PATH) FOR
+SELECT *
+FROM x, generate_series(1, x.a)
+WHERE x.a = 1
+----
+Explained Query:
+  Project (#0, #1, #0)
```

Comment:
Which transform is responsible for this optimization? The transform normally generates a `Map(1)` node here, right?

Do we have any testing infrastructure that can test a specific transform in isolation?